src/js/dropdown-menu/dropdown.underscore

```diff
@@ -1,5 +1,11 @@
-<a href="<%- main.url %>" class="user-title">
-    <span class="sr"><%- main.screenreader_label %> </span><%- main.text %>
+<a href="<%- main.url %>" class="menu-title">
+    <% if (main.screenreader_label) { %>
+        <span class="sr"><%- main.screenreader_label %> </span>
+    <% } %>
+    <% if (main.image) { %>
+        <img class="menu-image" src="<%- main.image %>" alt="">
```

Comment:
Please add these two new attributes to the docstring for the class, so they will be documented when we auto-generate the API doc site.